include/drivers/irq/riscv-plic.h

```diff
@@ -45,6 +45,12 @@
 #define PLAT_PLIC_THRES_ADJUST(x) ((x) - PLIC_THRES_PER_CONTEXT)
 #define PLAT_PLIC_EN_ADJUST(x)    ((x) - PLIC_EN_PER_CONTEXT)
 
+#elif defined CONFIG_PLAT_ARIANE
+
```

Comment:
Could you add some comments, pointing to the Ariane PLIC manual for future reference? The one on the Ariane github wiki seems outdated.